src/Client.php

```diff
@@ -92,12 +100,16 @@ public function deleteEvent(
 		}
 	}
 
+	/**
+	 * @param string[] $attendeesMails
+	 */
 	private function createEventModel(
 		string $subject,
 		DateTimeImmutable $start,
 		DateTimeImmutable $end,
 		bool $allDay = false,
-		?string $content = null
+		?string $content = null,
+		?array $attendeesMails = [],
```

Comment:
proč je to nullable? nebo jinak, proč není default null, když je to nullable? :-)